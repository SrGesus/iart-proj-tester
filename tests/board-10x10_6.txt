FE	VC	VB	FE	BC	FB	VD	LV	BB	FD
LH	FB	BE	VC	VD	LV	BE	FE	VB	VC
LV	VD	VD	VE	LV	VC	FC	LV	FD	LH
LH	VC	LV	VC	FE	VB	FD	LV	LV	LV
VE	BD	BC	VD	BE	BD	LV	VB	VE	BC
VC	BE	BE	BD	VC	FD	VD	LH	LV	VE
FC	LV	LH	VE	LV	LH	VB	FC	FD	VC
VC	VB	BD	BE	LV	BC	FC	BB	LH	BC
LH	FB	VB	LH	VD	VC	FB	VE	VE	LH
VE	FC	FC	VD	VB	LV	BE	LH	VC	FB
