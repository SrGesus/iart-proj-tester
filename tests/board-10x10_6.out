FB	VB	VE	FD	BB	FE	VB	LH	BB	FE
LV	FC	BD	VE	VD	LH	BE	FB	VD	VE
LV	VB	VC	VD	LH	VE	FC	LV	FB	LV
LV	VD	LH	VE	FB	VD	FE	LV	LV	LV
VD	BB	BB	VC	BD	BB	LH	VC	VD	BE
VB	BE	BD	BB	VC	FC	VB	LH	LH	VC
FC	LV	LV	VD	LH	LH	VC	FB	FD	VE
VB	VC	BD	BB	LH	BB	FE	BD	LH	BE
LV	FD	VC	LV	VB	VC	FB	VD	VE	LV
VD	FE	FD	VC	VD	LH	BC	LH	VC	FC
